src/emulator/config/src/config.cpp

```diff
@@ -180,6 +183,9 @@ ExitCode serialize_config(Config &cfg, const fs::path &output_path) {
     config_file_emit_single(emitter, "pref-path", cfg.pref_path);
     config_file_emit_vector(emitter, "lle-modules", cfg.lle_modules);
     config_file_emit_single(emitter, "discord-rich-presence", cfg.discord_rich_presence);
+    config_file_emit_single(emitter, "fps-limit", cfg.fps_limit;
```

Comment:
```suggestion
    config_file_emit_single(emitter, "fps-limit", cfg.fps_limit);
```
The `)` ran away.

Reply:
Thanks, forgot to check if the build passed on this.
As far as the sleep goes, it works perfectly fine in the homebrews I've tried. Still, might hit up the guys trying out various games to try and use the option for a bit.